app/models/move.rb

```diff
@@ -74,4 +74,8 @@ def rollable?
   def self.policy_class
     MovePolicy
   end
+
+  def class
```

Comment:
This did not appear to break any existing functionally but I'm not sure if this is the best way to handle this. This was added to work with `show_page_buttons`, without overriding `class` this may return the MoveType, ex: `Moves::Descriptive` which `link_to` would not be able to determine the path.